README.md

```diff
@@ -47,9 +47,9 @@ import Ember from 'ember';
 import { CanMixin } from 'ember-can';
 
 export default Ember.Route.extend(CanMixin, {
-  beforeModel: function() {
+  beforeModel() {
     if (!this.can('write post')) {
```

Comment:
I actually think super should always be called. For instance, what if some mixin model hook loads info about the user and their abilities (not a good pattern, just as an example).

I propose that this reads:

```js
beforeModel() {
  this._super(...arguments);

   if (!this.can('write post')) {
     return this.transitionTo('index');
  }
}
```